lib/iris/tests/unit/fileformats/grib/load_rules/test_convert.py

```diff
@@ -95,7 +95,8 @@ def assert_bounded_message(self, **kwargs):
                       'edition': 1, '_forecastTime': 15,
                       '_forecastTimeUnit': 'hours',
                       'phenomenon_bounds': lambda u: (80, 120),
-                      '_phenomenonDateTime': -1}
+                      '_phenomenonDateTime': -1,
+                      'table2Version': 9999}
```

Comment:
Yes, somewhere in the GRIB loader is `if grib.table2Version < 128:` which doesn't work with a `Mock` object (not exactly sorting, but related to comparison operators.)